src/entity/Human.php

```diff
@@ -356,7 +356,7 @@ public function applyDamageModifiers(EntityDamageEvent $source) : void{
 			&& ($this->inventory->getItemInHand() instanceof Totem || $this->offHandInventory->getItem(0) instanceof Totem)){
 
 			$compensation = $this->getHealth() - $source->getFinalDamage() - 1;
```

Comment:
Seems to me like it should be sufficient to remove the `- 1` here. Though it looks to have been intentionally put there.

Reply:
the ``- 1`` is there to set the player back to 1HP when the totem activates